ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -798,6 +798,27 @@ tests:
     - ref: openshift-extended-web-tests
     - ref: cucushift-e2e-report
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-kubevirt-hypershift-mce-guest
+- as: aws-ipi-ovn-kubevirt-hypershift-mce-guest-p3-f21-destructive
+  cron: 25 9 13 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      CNV_PRERELEASE_VERSION: "4.12"
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and @hypershift-hosted and
+        not @fips and @destructive'
+      TAG_VERSION: '@4.12'
+      TEST_ADDITIONAL: Hypershift
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~NonPreRelease&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&;Serial&;Disruptive&
```

Comment:
it should be `~ChkUpgrade&;~DisconnectedOnly&;~NonPreRelease&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&;Serial;Disruptive`

Reply:
@kuiwang02 Thanks a lot! Could you help to review it again?